packages/apps/dev-wallet/src/pages/backup-recovery-phrase/write-down/write-down-recovery-phrase.tsx

```diff
@@ -73,7 +73,7 @@ export function WriteDownRecoveryPhrase() {
         <Text size="small">{mnemonic}</Text>
         <Button
           type="submit"
-          onClick={() => {
+          onPress={() => {
```

Comment:
In my opinion, it's preferable to adhere more closely to the native browser APIs. Although these APIs predate touch events, we currently don't support a `press` event, which makes this counterintuitive. Moreover, using the component now requires learning its specific API, which feels redundant, especially for something as basic as a button. What was the reason moving towards `onPress`?

Reply:
This was documented in the shared type for the button & link:
``` typescript /**
   * @deprecated use `onPress` instead to be consistent with React Aria, also keep in mind that `onPress` is not a native event it is a synthetic event created by React Aria
   * @see https://react-spectrum.adobe.com/react-aria/useButton.html#props
   */
  onClick?: ComponentProps<'button'>['onClick'];

I agree on using onClick feels more intuitive for a web developer, this API design is coming from React Aria which we kind of extend.

Reply:
Ahh I missed this one. So this is something from React-Aria. I'm curious why Adobe chose this?

Reply:
I think they described it in their blog post about the button [link](https://react-spectrum.adobe.com/blog/building-a-button-part-1.html). In short they wanted to create a unified way to handle all interaction on multiple platforms.